openadapt/visualize.py

```diff
@@ -346,10 +346,13 @@ def main(
 
                 action_event_dict = row2dict(action_event)
                 window_event_dict = row2dict(action_event.window_event)
+                browser_event_dict = row2dict(action_event.browser_event)
 
                 if SCRUB:
                     action_event_dict = scrub.scrub_dict(action_event_dict)
                     window_event_dict = scrub.scrub_dict(window_event_dict)
+                    # browser_event_dict = scrub.scrub_dict(browser_event_dict)
```

Comment:
Why is this commented out?

Reply:
Done here: 98cdaa60d117d9e0e8197f8aad7a27660de73174

Yes,  I do have to uncomment it.